optuna/visualization/_slice.py

```diff
@@ -21,6 +24,82 @@
 _logger = get_logger(__name__)
 
 
+class _SliceSubplotInfo(NamedTuple):
+    param_name: str
+    x: List[Any]
+    y: List[float]
+    trial_numbers: List[int]
+    is_x_log_scale: bool
+    is_x_nunmerical: bool
+
+
+class _SlicePlotInfo(NamedTuple):
+    target_name: str
```

Comment:
It seems that the `_SlicePlotInfo` don't have to have `target_name`. It is specified in `_get_slice_plot` directly.